pandas/tests/io/excel/test_readers.py

```diff
@@ -557,10 +561,30 @@ def test_use_nullable_dtypes(self, read_ext):
         )
         with tm.ensure_clean(read_ext) as file_path:
             df.to_excel(file_path, "test", index=False)
-            result = pd.read_excel(
-                file_path, sheet_name="test", use_nullable_dtypes=True
+            with pd.option_context("io.nullable_backend", nullable_backend):
+                result = pd.read_excel(
+                    file_path, sheet_name="test", use_nullable_dtypes=True
+                )
+        if nullable_backend == "pyarrow":
+            import pyarrow as pa
+
+            from pandas.arrays import ArrowExtensionArray
+
+            expected = DataFrame(
+                {
+                    col: ArrowExtensionArray(pa.array(df[col], from_pandas=True))
+                    for col in df.columns
+                }
             )
-        tm.assert_frame_equal(result, df)
+            # pyarrow by default infers timestamp resolution as us, not ns
+            expected["i"] = ArrowExtensionArray(
+                expected["i"].array._data.cast(pa.timestamp(unit="us"))
+            )
+            # pyarrow supports a null type, so don't have to default to Int64
+            expected["j"] = ArrowExtensionArray(pa.array([None, None]))
```

Comment:
@phofl I noticed that in `_infer_types` that `result_mask.all()` (all nulls I think) would default the dtype to `Int64`. Any specific reason why Int64 was chosen?

Reply:
Yes, if you do:

```
data = """a,b
,,
,,"""

df = pd.read_csv(StringIO(data))

df.convert_dtypes()
```

you get back Int64 for all columns. Not saying this is perfect, but it made sense to model it after the existing behavior.

Reply:
Ah okay makes sense why this is the case then